docs/api.rst

```diff
@@ -1896,6 +1896,42 @@ Purging locally saved ethereum transactions
    :statuscode 500: Internal rotki error
 
 
+Decode transactions that haven't been decoded yet
+=================================================
+
+.. http:delete:: /api/(version)/blockchains/ETH/transactions/decode
+
+   Doing a POST on the transactions decoding endpoint will start the decoding process for all the transactions that haven't been decoded yet. Transactions already decoded won't be re-decoded.
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/blockchains/ETH/transactions/decode HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"async_query": false}
+
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      { "result": {"amount_decoded": 4}, "message": "" }
```

Comment:
Here and anywhere else you use it, please use `decoded_tx_number`.

Study the difference between `number` and `amount` in the english language to understand. It's a bit subtle.
https://www.enago.com/academy/understanding-the-difference-between-number-and-amount/#:~:text=What%20is%20the%20difference%2C%20then,things%20that%20cannot%20be%20counted.

```suggestion
      { "result": {"decoded_tx_number": 4}, "message": "" }
```